app/common/model/src/main/java/io/syndesis/common/model/integration/step/template/VelocityTemplatePreProcessor.java

```diff
@@ -61,6 +61,7 @@ protected boolean isText(String token) {
     }
 
     @Override
+    @SuppressWarnings("JdkObsolete") // j.u.Matcher accepts only StringBuffer
```

Comment:
Maybe we should add some issue on github to remind us to check about this at some point?

Reply:
Some of this can't be fixed easily, i.e. when using JDK classes that work with StrinBuffer and not StringBuilder...